.gitignore

```diff
@@ -1,5 +1,5 @@
-# PDF should be generated by ShareLaTeX
-*.pdf
+# PDF no longer generated by ShareLaTeX
```

Comment:
Suggestion:
PDF generated with ShareLatex and output copied to this repository manually.